src/insns/scbnds_32bit.adoc

```diff
@@ -42,6 +42,8 @@ tag is 0  or `cs1` is sealed.
 +
 `immediate = ZeroExtend(s ? uimm<<4 : uimm)`
 
+include::malformed_return_0.adoc[]
```

Comment:
Should this be "returns zero-bounds capability" or did you mean treat input as untagged?

Reply:
ok - done - please have another look

Reply:
There's something odd here: I think we want to say the bounds check fails. Otherwise a malformed cap should allow `SCBNDS` to address zero and length zero.